libbeat/scripts/Makefile

```diff
@@ -203,6 +203,16 @@ coverage-report:
 update:
 	bash ${ES_BEATS}/libbeat/scripts/update.sh ${BEATNAME} ${BEAT_DIR}/${BEATNAME} ${ES_BEATS}/libbeat
 
+### KIBANA FILES HANDLING ###
+
+KIBANA_ES?=http://localhost:9200/
+.PHONY: update
+kibana-export:
+	python ${ES_BEATS}/libbeat/scripts/kibana_export.py --url ${KIBANA_ES} --dir $(shell pwd)/etc/kibana
+
+.PHONY: update
+kibana-import:
```

Comment:
On second thought, I think it's ok to have python as dependency especially when it's used for development and we already use python for other stuff like the integration tests. @tsg @andrewkroh